model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -815,6 +815,7 @@ message StartWorkerRequest {
   org.apache.beam.model.pipeline.v1.ApiServiceDescriptor logging_endpoint = 3;
   org.apache.beam.model.pipeline.v1.ApiServiceDescriptor artifact_endpoint = 4;
   org.apache.beam.model.pipeline.v1.ApiServiceDescriptor provision_endpoint = 5;
+  string semi_persist_dir = 6;
```

Comment:
Should this be dynamic or rather configured up front for the worker pool?

Reply:
Great to have your suggestions. :)
Maybe I have not understood your idea. If we configure the dir for the whole pool, we 
may loose the flexibility that different jobs may set different semi_persist_dir for the workers in the same worker pool?

Reply:
I'm not sure whether this flexibility is desired. I could imagine that the person who starts the worker pool does not want arbitrary persist directories, but rather rather a fixed one.

Reply:
This should not be added here. Pipeline options are provided through the provisioning endpoint.

Reply:
Oh, Yes, I see, this is useless changes. `pipeline_options` already defined in the `ProvisionInfo`.